doc/reference.conf

```
@@ -1315,6 +1315,10 @@ general {
 	 */
 	ping_cookie = no;
 
+	/* ping warn time: how often to warn opers of an unresponsive server
+	 */
```

Maybe something like this?

```suggestion
	/* ping warn time: how long to wait after pinging a server before starting
	 * to complain it is unresponsive. Note that the ping check interval is 30 
	 * seconds, so the first complaint will come at the next check after this 
	 * time has passed.
	 */
```